build/lint.cloudbuild.yaml

```diff
@@ -11,3 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+steps:
+  - name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+    id: 'lint'
+    args: ['/usr/local/bin/test_lint.sh']
+tags:
+  - 'ci'
+  - 'lint'
+substitutions:
+  _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.5.2'
```

Comment:
```suggestion
  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0'
```

Reply:
after update have issue with Project factory
```
on .terraform/modules/project-ci-dataflow/terraform-google-modules-terraform-google-project-factory-f93d3cd/modules/core_project_factory/main.tf line 165, in resource "google_project_services" "project_services_authority":
165: resource "google_project_services" "project_services_authority" {

The provider provider.google does not support resource type
"google_project_services".
```

Reply:
it's actually not depends on image version, will discuss on a today call